pkg/aggregator/k8saggregator.go

```diff
@@ -128,10 +127,10 @@ func (a k8sAggr) Serve(ctx context.Context) error {
 					Message("Failed to parse worker's build ID.")
 				continue
 			}
-			if inProgress.Has(pod.UID) {
+			if !inProgress.Add(pod.UID) {
+				// Failed to add => Already beeing processed
```

Comment:
typ: `beeing` -> `being`
```suggestion
				// Failed to add => Already being processed
```